lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+
+  def initialize
+    @dishes = []
+  end
+
+  def add(dish)
+    @dishes << dish
+  end
+
+  def choose(description)
+    @dishes.find {|dish| dish.description == description}
+  end
+
+  def dishes
+    @dishes.dup
+  end
+
+  def to_s
+    @dishes.inject(String.new) {|memo, dish| memo << "#{dish.to_s}\n" }
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (https://github.com/bbatsov/ruby-style-guide#no-to-s)<br>Style/EmptyLiteral: Use string literal `''` instead of `String.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)